exporter/collector/logs.go

```diff
@@ -393,8 +393,15 @@ func (l logMapper) logToSplitEntries(
 
 	// parse LogEntrySourceLocation struct from OTel attribute
 	if sourceLocation, ok := attrsMap[SourceLocationAttributeKey]; ok {
+		var sourceLocationBytes []byte
```

Comment:
Maybe worth refactoring this into a common function that's used by both call sites? (Especially since it's going to be needed a third time to support `operation`)

Reply:
Done in [a0f37b2](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/794/commits/a0f37b2a6901f8c2529db678a696f414e0a6bf5b)